src/languages/en.js

```diff
@@ -126,44 +137,71 @@ export default {
     addSecondaryLoginPage: {
         addPhoneNumber: 'Add Phone Number',
         addEmailAddress: 'Add Email Address',
-        enterPreferredPhoneNumberToSendValidationLink: 'Enter your preferred phone number and password to send a validation link.',
-        enterPreferredEmailToSendValidationLink: 'Enter your preferred email address and password to send a validation link.',
+        enterPreferredPhoneNumberToSendValidationLink:
+            'Enter your preferred phone number and password to send a validation link.',
+        enterPreferredEmailToSendValidationLink:
+            'Enter your preferred email address and password to send a validation link.',
         sendValidation: 'Send Validation',
-
     },
     initialSettingsPage: {
         settings: 'Settings',
+        about: 'About',
+        aboutPage: {
+            appDownloadLinks: 'App Download Links',
```

Comment:
I think this should be `App download links` so it follows the same capitalisation format as the others. @trjExpensify how do you feel about that?

Reply:
Dome :)

Reply:
Agreed 👍